help/en/html/tools/Sensors.shtml

```diff
@@ -157,51 +111,83 @@
+            sensor.  If a hardware address already exists in the Sensors range that is being added, then JMRI will skip
+            that address. If you have entered a User Name, then a number will be appended for each additional sensor to
+            ensure that the User Name is unique.
           </li>
 
           <li>
-            <p>Click the <b>OK</b> button at the bottom of the
+            Click the <strong>OK</strong> button at the bottom of the
             pane. If everything is fine, a message stating "New
             Sensor added..." will be displayed in the notes area.
             If there is trouble with anything, an error or warning
             message will be displayed in the notes area; you should
-            then correct the error and click <b>OK</b> again.</p>
+            then correct the error and click <strong>OK</strong> again.
           </li>
 
           <li>
-            <p>Back in the main Sensor Table, you can then see the
-            current status of the Sensor.</p>
+            You will then see the current status of the sensor (or sensors) in the main Sensor Table pane.
           </li>
         </ol>
         
-        <h3>Sensor Groups</h3>
-        <p>A group of sensors of which at most one can be active.
-        When a member of the group becomes active, all others are set to inactive.</p>
+        <p>For additional information about working with the Sensor Table,
+        <a href="../../package/jmri/jmrit/beantable/SensorTable.shtml">click here.</a>
+
+        <h3>Sensor Table Controls</h3>
+
+        <p>Below the table are check boxes to display additional columns with sensor control information.</p>
+
+        <p>The <strong>Defaults</strong> pulldown menu allows you to set two system-wide
+        settings:</p>
+
+        <ul>
+          <li><a href="#Debounce">Global Debounce Delay</a></li>
+
+          <li>Initial Sensor State (for Internal Sensors only): lets you choose what state Internal Sensors (system name beginning with
+       "IS") will display after JMRI startup. [Note that the initial state of other sensors cannot be set directly by JMRI, 
+       although you can use a <a href="scripting/index.shtml">script</a> to set them.</li>
+        </ul>
+        
+        <h2>Sensor Groups</h2>
         
-        <p>Sensor Groups are configured via the <a 
-            href="../../package/jmri/jmrit/sensorgroup/SensorGroupFrame.shtml"
-            >Sensor Group Tool</a>.</p>
+        <p>A Sensor Group is a group of sensors of which at most one can be active at a time.  When one member of the group
+        becomes active, all others are set to inactive.</p>
+        
+        <p>Sensor Groups are configured via the <a href="../../package/jmri/jmrit/sensorgroup/SensorGroupFrame.shtml">
+        Sensor Group Tool</a>.</p>
+            
+        <h2><a name="Debounce" id="Debounce"></a>Sensor Debounce Delay</h2>
+
+        <p>The Sensor Debounce Delay facility is there to help prevent
```

Comment:
The repositioning away from the discussion of sensors themselves, plus calling this a "faclity" makes this sound like something apart from the Sensors themselves.  It's not.